sktime/transformers/resizing.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import pandas as pd
+from scipy import interpolate
+
+from sktime.transformers.base import BaseTransformer
+from sktime.utils.validation.supervised import validate_X
+
+
+class TSResizeTransform(BaseTransformer):
+    """Transformer that rescales series for another number of points. 

```

Comment:
Please update doc string:
* ensure you follow numpy/scikit-learn formatting
* add descriptions what happens when series is longer/shorter than length from your comment on PR
* the transformer doesn't "rescale" the series, it rather interpolates/subsamples it, right?